src/main/java/com/ldtteam/structurize/client/gui/WindowTagTool.java

```diff
@@ -200,4 +226,31 @@ public void updateElement(final int index, final Pane rowPane)
             }
         });
     }
+
+    /**
+     * Updates the current tag list
+     */
+    public void updateTagOptionList()
+    {
+        tagOptionList.setDataProvider(new ScrollingList.DataProvider()
+        {
+            @Override
+            public int getElementCount()
+            {
+                return TAG_OPTIONS.size();
+            }
+
+            @Override
+            public void updateElement(final int index, final Pane rowPane)
+            {
+                final Text tagsText = rowPane.findPaneOfTypeByID(TAG_TEXT, Text.class);
+                tagsText.setText(Component.literal(TAG_OPTIONS.get(index)));
+                PaneBuilders.tooltipBuilder().hoverPane(tagsText).build()
+                    .setText(Component.translatable("com.ldtteam.tag.tooltip." + TAG_OPTIONS.get(index)));
```

Comment:
```suggestion
                    .setText(Component.translatable("structurize.gui.tagtool.tooltip." + TAG_OPTIONS.get(index)));
```

Reply:
well, i rather use modid in strings so it's always clean who is the owner

Reply:
Yes, but here the owner is not structurize. It's the mod that adds the tag option. That mod will add it to their lang file too.

Reply:
the owner is structurize, cuz without it there is not lang string :D